src/core/MOM_forcing_type.F90

```diff
@@ -592,21 +595,21 @@ subroutine extractFluxes1d(G, GV, US, fluxes, optics, nsw, j, dt, &
     ! net volume/mass of liquid and solid passing through surface boundary fluxes
     netMassInOut(i) = dt * (scale * &
                                    (((((( fluxes%lprec(i,j)        &
-                                        + fluxes%fprec(i,j)      )  &
-                                        + fluxes%evap(i,j)       )  &
-                                        + fluxes%lrunoff(i,j)    )  &
-                                        + fluxes%vprec(i,j)      )  &
-                                        + fluxes%seaice_melt(i,j))  &
-                                        + fluxes%frunoff(i,j)    ))
+                                         + fluxes%seaice_melt(i,j))  &
```

Comment:
Both this reordering of the math and the one in about another 10 lines are needed to preserve answers with the FMS coupler, but will change answers with the NUOPC coupler.  A runtime logical flag might be needed here to preserve both sets of answers, and make the default the setting that is needed with the NUOPC coupler, since active intervention will be needed to use seaice_melt with the FMS coupler.